recursive_count_th/count_th.py

```diff
@@ -8,17 +8,16 @@
 
 def count_th(word):
 
-    count = 0
     if word == word.count('th'):
-        return 
-        count += 1
-
-    return word.count('th')
+        return 1
+    else:
+        return word.count('th')
```

Comment:
In python, `.count()` is still a loop underneath the hood. It's kinda like `.filter()`, `.map()`, or `.find()` in JS - you don't actually write a for loop itself, but there is a loop going underneath the hood.